integration/tests/server/scenarios/system_scenario.rs

```diff
@@ -124,8 +125,8 @@ pub async fn run(client_factory: &dyn ClientFactory) {
     assert_eq!(topic.compression_algorithm, CompressionAlgorithm::default());
     assert_eq!(topic.size, 0);
     assert_eq!(topic.messages_count, 0);
-    assert_eq!(topic.message_expiry, None);
-    assert_eq!(topic.max_topic_size, None);
+    assert_eq!(topic.message_expiry, IggyExpiry::NeverExpire);
+    assert_eq!(topic.max_topic_size, MaxTopicSize::get_server_default());
```

Comment:
The reason is that the `MaxTopicSize::ServerDefault` has a value of `0` and when used in the command, given that the server configuration doesn't specify any particular size, it will set the actual default value of 10 GB for max topic size.